backend/src/routes/api/status/statusUtils.ts

```diff
@@ -35,6 +36,13 @@ export const status = async (
       const adminUsers = await getGroup(customObjectsApi, adminGroup);
       isAdmin = adminUsers?.includes(userName) ?? false;
     }
+
+    if (allowedGroup === '') {
+      throw new Error('It is not allowed to set an empty string as allowed group.');
+    } else {
+      const allowedUsers = await getGroup(customObjectsApi, allowedGroup);
+      isAllowed = allowedUsers?.includes(userName) ?? false;
```

Comment:
`isAllowed` is false, but `isAdmin` is true... that's confusing, no? We should auto grant `isAllowed` when they are admins, right?

Reply:
Fixed.